server/testkit/tests/integration.rs

```diff
@@ -219,3 +227,135 @@ async fn test_idm_domain_set_ldap_basedn(rsclient: KanidmClient) {
         .await
         .is_err());
 }
+
+/// run a test command as the admin user
+fn test_cmd_admin(token_cache_path: &str, rsclient: &KanidmClient, cmd: &str) -> Output {
+    let split_cmd: Vec<&str> = cmd.split_ascii_whitespace().collect();
+    test_cmd_admin_split(token_cache_path, rsclient, &split_cmd)
+}
+/// run a test command as the admin user
+fn test_cmd_admin_split(token_cache_path: &str, rsclient: &KanidmClient, cmd: &[&str]) -> Output {
+    println!(
+        "##################################\nrunning {}\n##################################",
+        cmd.join(" ")
+    );
+    let res = cli_kanidm!()
+        .env("KANIDM_PASSWORD", ADMIN_TEST_PASSWORD)
+        .args(cmd)
+        .output()
+        .unwrap();
+    println!("############ result ##################");
+    println!("status: {:?}", res.status);
+    println!("stdout: {}", String::from_utf8_lossy(&res.stdout));
+    println!("stderr: {}", String::from_utf8_lossy(&res.stderr));
+    println!("######################################");
+    assert!(res.status.success());
+    res
+}
+
+/// run a test command as the idm_admin user
+fn test_cmd_idm_admin(token_cache_path: &str, rsclient: &KanidmClient, cmd: &str) -> Output {
+    println!("##############################\nrunning {}", cmd);
+    let res = cli_kanidm!()
+        .env("KANIDM_PASSWORD", IDM_ADMIN_TEST_PASSWORD)
+        .args(cmd.split(" "))
+        .output()
+        .unwrap();
+    println!("##############################\n{} result: {:?}", cmd, res);
+    assert!(res.status.success());
+    res
+}
+
+#[kanidmd_testkit::test(threads = 4)]
```

Comment:
Should we be hardcoding this value?

Reply:
Lemme try removing it, I know I had to set it at one point because something blew up

Reply:
yep, all gud